lib/vec.h

```diff
@@ -92,7 +92,7 @@ struct m0_vec_cursor {
 	/** Segment that the cursor is currently in. */
 	uint32_t             vc_seg;
 	/** Offset within the segment that the cursor is positioned at. */
-	m0_bindex_t          vc_offset;
```

Comment:
I think we should keep  the original code.
This is offset, and offset is an index in the object.

Reply:
Offset is on how much to shift, isn't it? So it's a count, not an index.